scipy/stats/tests/test_distributions.py

```diff
@@ -3236,6 +3236,48 @@ def test_logsf(self):
         y = stats.rayleigh.logsf(50)
         assert_allclose(y, -1250)
 
+    @pytest.mark.parametrize("floc", [-1, 0, 1, 2, 3])
+    @pytest.mark.parametrize("fscale", [.5, 1, 2, 3, 4])
+    def test_fit(self, floc, fscale):
+        data = stats.rayleigh.rvs(size=250, loc=floc, scale=fscale)
+
+        def scale_mle(data, floc):
+            return (np.sum((data - floc) ** 2) / (2 * len(data))) ** .5
+
+        scale_expect = scale_mle(data, floc)
+        # when `floc` is provided, direct MLE equation is used
+        loc, scale = stats.rayleigh.fit(data, floc=floc)
+        assert_equal(loc, floc)
+        assert_equal(scale, scale_expect)
```

Comment:
```suggestion
        assert_equal(scale, scale_expect)
        
```

Reply:
I was suggesting that you add a line break here to separate distinct sections of the test. Same above; I'd add that line break back to separate the first section of the test from the definition of `scale_mle`, which is also used in other sections.

Reply:
Code snipped doesn't say outdated, but a line break has been added here and for [this comment](https://github.com/scipy/scipy/pull/12097/files#r463845257.)